trillian/ctfe/services.go

```diff
@@ -82,6 +105,118 @@ func (s *issuanceChainService) Add(ctx context.Context, chain []byte) ([]byte, e
 	return hash, nil
 }
 
+// BuildLogLeaf builds the MerkleTreeLeaf that gets sent to the backend, and make a trillian.LogLeaf for it.
```

Comment:
This does more than building the log leaf, it also stores the chain in the backend storage service, if any. Could you explain this please? Should this function be called "FixLogLeaf" instead?

Reply:
I don't have a better idea for this method name. There are already code comments in the next following lines to explain the storage. I can add a similar sentence in the method comment but that seems to be duplicated. `FixLogLeaf` assumes there is already a LogLeaf struct as the method argument to fix.